api/management/api.go

```diff
@@ -584,30 +584,39 @@ func (rac RealmAdminConfiguration) Validate() error {
 	return validation.NewParameterValidator().
 		ValidateParameterIn("mode", rac.Mode, allowedAdminConfMode, true).
 		ValidateParameterFunc(rac.validateAvailableChecks).
-		ValidateParameterFunc(rac.validateAccreditations).
 		Status()
 }
 
 func (rac RealmAdminConfiguration) validateAvailableChecks() error {
+	var accredConditions, err = rac.validateAccreditations()
+	if err != nil {
+		return err
+	}
+
 	if len(rac.AvailableChecks) > 0 {
```

Comment:
Est ce que le check sur length est vraiment nécessaire. Si len == 0 , le for ne vas pas tourner non ?

Reply:
J'ai fait des tests... A priori, pas de souci... j'ai viré les checks de len(...)==0